packages/form-core/test-suites/choice-group/ChoiceInputMixin.suite.js

```diff
@@ -95,6 +95,21 @@ export function runChoiceInputMixinSuite({ tagString } = {}) {
       expect(counter).to.equal(1);
     });
 
+    it('adds "isTriggerByUser" flag on model-value-changed', async () => {
+      let isTriggeredByUser;
+      const el = /** @type {ChoiceInput} */ (await fixture(html`
+      <${tag}
+        @model-value-changed="${(/** @type {CustomEvent} */ event) => {
+          isTriggeredByUser = event.detail.isTriggeredByUser;
+        }}"
+      >
+        <input slot="input" />
+      </${tag}>
+    `));
+      el._inputNode.dispatchEvent(new CustomEvent('change', { bubbles: true }));
+      expect(isTriggeredByUser).to.be.true;
```

Comment:
```suggestion
      expect(isTriggeredByUser).to.be.false;
```
Since it's dispatched programmatically? Or does this fall under "mimicking user input" and therefore we need `isTriggeredByUser` metadata, because `isTrusted` is false?

Reply:
This can be seen as mimicing user input, because we fire an event that normally would have been fired by the platform.
Setting it programmatically (via .modelValue or .checked) should return in false